### ISSUE TITLE
Disjoint batch data GCN

### ISSUE THREAD

**Issue by nikita8** [CSCCNY/final-project-faultdetection-cnny#5]
Prepare disjoint batch data to feed into the GCN model such that many graphs are considered during the model training.

**Comment by nikita8** [comment]
- Used batch normalization for input and intermediate GCN outputs
- Used 5% dropout rate
- Computed class losses weighted by the number of instances in the class
- Adjusted weights further by 60:40 ratio
